media-gfx/alembic/alembic-1.7.9.ebuild

```diff
@@ -0,0 +1,119 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+
+inherit cmake-utils python-single-r1
+
+DESCRIPTION="Alembic is an open framework for storing and sharing scene data"
+HOMEPAGE="http://alembic.io/"
```

Comment:
how about `https`?

Reply:
does not work for alembic.io, but for www.alembic.io. Changed it.